doc/administrator_guide.md

```diff
@@ -1,6 +1,10 @@
-Administrator guide
-===================
-
+---
+title: "Administrator guide"
+date: 2021-11-18
+draft: false
+wieght: 40
+description: "Deploying, maintaining, and upgrading a KCIDB service installation"
+---
 Kcidb infrastructure is mostly based on Google Cloud services at the moment:
 
     === Hosts ===  ======================= Google Cloud Project ========================
```

Comment:
ASCII art is cool, but for this static website there's also [Mermaid](https://mermaid-js.github.io/mermaid/#/) which is easier on the eyes.

See this example:
https://github.com/kernelci/kernelci-project/blame/main/kernelci.org/content/en/docs/architecture.md#L53
and how it's rendered:
https://kernelci.org/docs/architecture/

Reply:
Neat! Gotta learn that, thank you. For now though, let's start with the ASCII art :grin:

Reply:
Posted an issue to remember: https://github.com/kernelci/kcidb/issues/240